examples/material-ui-cra-styled-components-ts/src/App.tsx

```diff
@@ -21,7 +21,7 @@ export default function App() {
   return (
     <Container maxWidth="sm">
       <Box sx={{ my: 4 }}>
-        <Typography variant="h4" component="h1" gutterBottom>
+        <Typography variant="h4" component="h1" sx={{ mb: 2 }}>
```

Comment:
We might not want to promote `gutterBottom` much. I recall adding this for markdown like formatting, it's arbitrary: https://github.com/mui/material-ui/issues/16926. I don't know if it's idiomatic anymore.